dev_guide/deployments.adoc

```diff
@@ -763,3 +763,21 @@ Labels can only be set to one value, so setting a node selector of `region=west`
 in a pod configuration that has `region=east` as the administrator-set default,
 results in a pod that will never be scheduled.
 ====
+
+== Running a Pod with a Different Service Account
+
+You can run a pod with a service account other than the default. To specify an alternate service account, modify the `*spec*` field of the deployment configuration:
```

Comment:
We're repeating "edit the config". Maybe get rid of it above and leave it as the first sentence? The fact that there's a procedure upcoming speaks for itself.